mkosi

```diff
@@ -2719,12 +2736,13 @@ def calculate_signature(args: CommandLineArguments, checksum: Optional[IO[Any]])
     return f
 
 
-def calculate_bmap(args: CommandLineArguments, raw: BinaryIO) -> Optional[TextIO]:
+def calculate_bmap(args: CommandLineArguments, raw: Optional[BinaryIO]) -> Optional[TextIO]:
     if not args.bmap:
         return None
 
     if not args.output_format.is_disk_rw():
         return None
+    assert raw is not None
```

Comment:
I wonder if there should be a blank line before this addition. Usually, the `assert … is not None` is right after the single `if …: return` check, but here we have two such checks separated by a blank line, and the `assert` doesn’t really belong any closer to the second one than the first one, in my opinion.

Reply:
I know that `is_disk_rw()` implies `is_disk()` implies `raw is not None`.

I'm not sure what the relationship between `bmap` and `raw` is; in my understanding, it is related the the second one, and not to the first.